recipe/meta.yaml

```diff
@@ -31,14 +31,11 @@ requirements:
 
 test:
   files:
-    - ice_skating.py
-    - male1.txt
-    - import_test.py
-    - test.sh
+    - tests/
```

Comment:
This has nothing to do with the failure. It only helps keep the feedstock organized.